frontend_lib/src/TracimTableLib/TracimTable/TracimTable.jsx

```diff
@@ -0,0 +1,170 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import {
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getSortedRowModel,
+  useReactTable
+} from '@tanstack/react-table'
+
+import EmptyListMessage from '../../component/EmptyListMessage/EmptyListMessage.jsx'
+import FilterBar from '../../component/FilterBar/FilterBar.jsx'
+import classnames from 'classnames'
+import Icon from '../../component/Icon/Icon.jsx'
+
+export const GenericTracimTableLine = props => {
+  return (
+    <div
+      className={`tracimTable__body__row ${props.customRowClass}`}
+      key={props.rowData.id}
+    >
+      {props.rowData.getVisibleCells().map(cell => (
+        <div
+          className={`tracimTable__body__row__cell ${cell.column.columnDef.className}`}
+          key={cell.id}
+        >
+          {flexRender(
+            cell.column.columnDef.cell,
+            {
+              ...cell.getContext()
+            }
+          )}
+        </div>
+      ))}
+    </div>
+  )
+}
+
+const TracimTable = (props) => {
+  const globalFilterFn = (row, columnId, value, addMeta) => {
+    return table.getColumn(columnId).getFilterFn()(row, columnId, value, addMeta)
+  }
+
+  const table = useReactTable({
+    data: props.data,
+    globalFilterFn,
```

Comment:
Just a suggestion to avoid abbreviations, if you prefer the other way, you can just not apply and resolve this conversation

```suggestion
  const handleGlobalFilter = (row, columnId, value, addMeta) => {
    return table.getColumn(columnId).getFilterFn()(row, columnId, value, addMeta)
  }

  const table = useReactTable({
    data: props.data,
    globalFilterFn: handleGlobalFilter,
```